SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -267,27 +268,31 @@
   <!-- The FxCop targets are conditionally imported so we need to make sure the required condition
        is true early enough. -->
   <PropertyGroup Condition=" $(SonarQubeTempPath) != '' ">
-    <SonarQubeRulesetFileName Condition=" $(SonarQubeRulesetFileName) == ''">SonarQubeAnalysis.ruleset</SonarQubeRulesetFileName>
```

Comment:
This has been discussed with @duncanpMS - we need at a very early stage to force the conditional imports of the FxCop targets.

However, it is so early that `$(ProjectLanguage)` is not available yet.

So now, we will always force the FxCop targets to be imported during a SonarQube analysis - but right before its task execution we can override a property to actually skip it.